src/traces/pie/plot.js

```diff
@@ -164,36 +169,39 @@ function plot(gd, cdModule) {
                     } else {
                         transform = transformInsideText(textBB, pt, cd0);
                         if(textPosition === 'auto' && transform.scale < 1) {
-                            sliceText.call(Drawing.font, trace.outsidetextfont);
-                            if(trace.outsidetextfont.family !== trace.insidetextfont.family ||
-                                    trace.outsidetextfont.size !== trace.insidetextfont.size) {
+                            var newFont = Lib.extendFlat({}, trace.outsidetextfont, {});
+                            newFont.size = Math.max(newFont.size, fullLayout.uniformtext.minsize || 0);
+
+                            sliceText.call(Drawing.font, newFont);
+                            if(newFont.family !== font.family || newFont.size !== font.size) {
```

Comment:
We shouldn't need this check. The caching mechanism in `Drawing.bBox` should be able to catche cases where `sliceText` has the same font family and size as in the previous call.

Reply:
Good call. I'll give it a try.

Reply:
Removed in https://github.com/plotly/plotly.js/pull/4420/commits/a27dc29ca87303fd53bf994bfe1203b3c5e96887.

Reply:
Great - I'm glad that worked out!